packages/ecc-utils-design/src/components/form/form.ts

```diff
@@ -190,8 +191,13 @@ export default class Form extends LitElement {
     }
 
     if (!_.get(this.form, path)) {
-      _.set(this.form, path, field.fieldOptions?.default || "");
+      if (field.fieldOptions?.default && !this.hasUpdated) {
+        _.set(this.form, path, field.fieldOptions.default);
+      } else if (field.fieldOptions?.returnIfEmpty) {
+        _.set(this.form, path, "");
```

Comment:
aren't you still setting it to empty str

Reply:
That's is returnIfEmpty is true